osv2/bulk.go

```diff
@@ -164,6 +197,11 @@ type BulkResponse struct {
 	Error  *Error                           `json:"error,omitempty"`
 }
 
+type BulkResponseOnlyErrors struct {
```

Comment:
Don't export this type since It's an implementation detail.  In fact, I would declare this locally inside of `parseResponse()` so that there's no explicit named type.